boule/_realizations.py

```diff
@@ -7,13 +7,57 @@
 """
 Ellipsoid and Sphere realizations for the Earth and other planetary bodies.
 """
+import numpy as np
+
 from ._ellipsoid import Ellipsoid
 from ._sphere import Sphere
 from ._triaxialellipsoid import TriaxialEllipsoid
 
+Mercury2015 = Sphere(
+    name="Mercury2015",
+    long_name="Mercury spheroid (2015)",
+    radius=2_439_372,
+    geocentric_grav_const=22.031839224e12,
+    angular_velocity=1.2400172589e-6,
+    reference=(
+        "Wieczorek, MA (2015). 10.05 - Gravity and Topography of the Terrestrial "
+        "Planets, Treatise of Geophysics (Second Edition); Elsevier. "
+        "doi:10.1016/B978-0-444-53802-4.00169-X"
+    ),
+)
+
+Mercury2024 = Sphere(
+    name="Mercury2024",
+    long_name="Mercury spheroid (2023)",
```

Comment:
Why in the name we have 2024 and here 2023? Is that a typo? Or is it correct?

Reply:
thanks! typo...